src/frontend/src/pages/WorkPackageDetailPage/WorkPackageViewContainer/WorkPackageViewContainer.tsx

```diff
@@ -167,7 +167,9 @@ const WorkPackageViewContainer: React.FC<WorkPackageViewContainerProps> = ({
       <HorizontalList
         title={'Blocked By'}
         items={workPackage.blockedBy.map((dep) => (
-          <strong>{wbsPipe(dep)}</strong>
```

Comment:
I think you should leave it as bold, just cause that makes it pop a little more, and we're trying to make dependencies more meaningful